### ISSUE TITLE
The title of every page always seems to be Data Skeptic

### ISSUE THREAD

**Issue by yaph** [kylepolich/dataskeptic#1]
Hi Kyle,

I just learned about your podcast and listened to a few episodes. Very interesting stuff. When I bookmarked the next show I want to listen to, I noticed that the title is "Data Skeptic" and not for example "Easily Fooling Deep Neural Networks".

I suggest you add the episode title to the title tag in the head section in https://github.com/kylepolich/dataskeptic/blob/master/www/header.php as it is more user friendly and also a factor for search engines when ranking results for the given keywords entered by the user.

**Comment by kylepolich** [comment]
Wonderful suggestion.  Sorry it took me so long to implement, but I've made this great change now.  Thanks!

**Comment by yaph** [comment]
No problem! Just looked at the podcast homepage and noticed that it has "Crypto" as its title now. Don't know where that comes from, but I guess it is not intended.

**Comment by kylepolich** [comment]
Indeed you are correct.  Thanks for the continued QA assistance!  I've fixed it now

In case you're curious, this was a logical condition I overlooked related to episode 64, whose title is "Crypto" and has the distinction of being the only episode (so far) that has a URL that ends in a directory (i.e. "/") which asks apache2 to default to index.*, which is also the default for dataskeptic.com/ and no other pages.

Thanks for bringing that case to my attention, I was able to correct quickly thanks to your help.
